packages/core-sdk/test/unit/resources/ipAccount.test.ts

```diff
@@ -217,4 +221,45 @@ describe("Test IPAccountClient", () => {
       expect(result).to.equal(txHash);
     });
   });
+
+  describe("Test transferErc20", () => {
+    it("should throw error when call transferErc20 failed", async () => {
```

Comment:
maybe one more test case, when one of the target token address is invalid, `executeBatch` should never be called and method should throw.